jupyterhub/auth.py

```diff
@@ -100,41 +101,74 @@ class Authenticator(LoggingConfigurable):
         """
     ).tag(config=True)
 
-    whitelist = Set(
+    whitelist = Set(help="Deprecated, use `Authenticator.allowed`", config=True,)
+
+    allowed = Set(
```

Comment:
Some authenticators have additional properties such as `allowed_groups`. Any thoughts on changing this to `allowed_users` for clarity?

Reply:
Good idea. Switched to `allowed_users` pending resolution of [the discussion on team-compass](https://github.com/jupyterhub/team-compass/issues/299#issuecomment-648759244)